src/components/cursor.js

```diff
@@ -64,9 +66,18 @@ module.exports.Component = registerComponent('cursor', {
     this.canvasBounds = document.body.getBoundingClientRect();
     this.isCursorDown = false;
 
+    // expose camera and cursor to user if required.
+    if (this.data.camera === 'user') {
+      this.camera = this.el.sceneEl.camera;
+    }
+    if (this.data.canvas === 'user') {
```

Comment:
This API is strange. Public variables are set or not depending on the value of a component property. Purpose is not clear either reading the schema or the docs. Probably feels to ad-hoc to the need of your use case.

Reply:
Yes, it is a bit ad hoc.

The reason for adding this to the interface was a desire to maintain complete back compatibility.

An example of a problem that could be caused by not putting this control on the interface is the following:

- An A-Frame app that switches between 2 different cameras, e.g. orthographic & perspective.
- With the existing cursor, cursor will auto-adapt to the current camera (as of[ this fix](https://github.com/aframevr/aframe/pull/4936)).
- After my change, it would become necessary to manually update the cursor's view of the camera when switching between ortho & perspective.
- I didn't want to risk breaking existing applications, hence the default setting of "auto" and the new mode needing to be explicitly activated in config.
- Not strictly necessary for 2 different parameters, but they are logically different functions so it seemed better.

In terms of the documentation, I think it would help a lot if I had an example to link to.  As per notes above, I'm hoping to re-use the component I have built as an example, but still TBD whether the owners of that code will be willing to open source it.

In short, I think a config option is necessary if we are to avoid breaking existing apps.  OTOH, the fix to make cursor work with an orthographic camera isn't even released yet (in master but not 1.2.0), so another option might be to simply document that fact that if you want to switch which camera is used for a cursor, you need to explicitly update `this.camera` in the cursor component.